readthedocs/templates/search/elastic_search.html

```diff
@@ -25,6 +25,8 @@
 
 {% block content %}
 
+{% trans "100" as MAX_SUBSTRING_LIMIT %}
```

Comment:
I don't think we need `trans` here, I think we can use `with`: https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#with